instruments/models.py

```diff
@@ -269,15 +272,24 @@ def to_topic_dict(self, language="en"):
             concept_key="concept_%s" % concept_name,
         )
 
-    def comparison_string(self, to_string=False):
-        cs = ["Question %s" % self.title()]
+    def comparison_string(self, to_string=False, wrap=50):
+        cs = ["Question: %s" % self.title()]
         for item in self.get_source().get("items", []):
             cs += [
                 "",
-                "Item: %s (scale: %s)" % (item.get("value"), item.get("scale")),
+                "Item: %s (scale: %s)" % (item.get("item"), item.get("scale")),
                 item.get("text", ""),
             ]
             cs += ["%s: %s" % (a["value"], a["label"]) for a in item.get("answers", [])]
+        if wrap:
+            cs_temp = [ textwrap.wrap(line, wrap) for line in cs ]
+            cs = []
+            for wrap in cs_temp:
```

Comment:
This "wrap" is not the same as the keyword argument of the method. It gets confusing if you say "if wrap" twice in a method, but the tested variable is actually a different thing. Can you change one of the variable names?

Reply:
@afuetterer Done.